haskell/haskell.bzl

```diff
@@ -150,6 +150,39 @@ timeout and resource allocation for the test.
 [bazel-test-attrs]: https://docs.bazel.build/versions/master/be/common-definitions.html#common-attributes-tests
 """
 
+def haskell_criterion(
+    tags = [],
+    args = [],
+    size = "enormous",
+    **kwargs
+  ):
+  """Build a benchmark suite using [criterion][criterion].
+
+This rule accepts the same arguments as [haskell_test][haskell_test_rule]
+The resulting benchmark is internally considered as a test because bazel has
+no primitive notion of benchmark.
+
+[criterion]: http://www.serpentine.com/criterion/
+[haskell_test_rule]: /haskell/haskell.html#haskell_test
+  """
+  default_criterion_args = \
+    [
+     "--csv=result.csv",
+     "--json=result.json",
+     "--output=result.html",
+     "--junit=result.xml",
```

Comment:
It must be important to know how to access these output files, that should go into the tutorial as well.